development/build/utils.js

```diff
@@ -29,6 +29,43 @@ function isTestBuild(buildTarget) {
   );
 }
 
+/**
+ * Extract the actual build target from a task name.
+ *
+ * Task names follow patterns like:
+ * - 'scripts:core:dev:standardEntryPoints' -> 'dev'
+ * - 'scripts:core:test:contentscript' -> 'test'
+ * - 'scripts:core:test-live:sentry' -> 'testDev'
+ * - 'test' -> 'test' (already a build target)
+ *
+ * @param {string} taskName - The task name or build target.
+ * @returns {BUILD_TARGETS | string} The extracted build target.
+ */
+function getBuildTargetFromTask(taskName) {
```

Comment:
Child processes receive task names (e.g., `scripts:core:test:standardEntryPoints`) instead of build targets (`test`). This function extracts the correct build target from task names so `getEnvironment()` works correctly in child processes.